language/go/config.go

```diff
@@ -59,10 +59,25 @@ type goConfig struct {
 	// depMode determines how imports that are not standard, indexed, or local
 	// (under the current prefix) should be resolved.
 	depMode dependencyMode
+
+	// goProtoCompilers is the protocol buffers compiler(s) to use for go code.
```

Comment:
Added directives.  It seems slightly odd that directive is plural (`go_grpc_compilers`) while the flag is singular (`go_grpc_compiler`), but I think it'd be equally weird if the flag were plural, since you're expected to pass it multiple times to get multiple values.

Reply:
I put the `splitValue` helper in the go-specific `config.go` file, since I wasn't sure how general purpose it'd be; perhaps for other directives comma-separated lists might not work?  Happy to move it (and the test) to `rule/directives.go` if you think that makes more sense.

Reply:
About the names, I think they make sense as they are. I guess the alternatives would be to make the command line flag plural an accept a list (escaping problems) or make the directive singular, specified multiple times (ambiguity between setting and appending).

Reply:
About `splitValue`, let's keep it in `language/go` for now. Agreed though that it's generally useful, and we may want it to be a public function in `rule` some day.